Darwin/StitchCore/StitchCore/StitchCoreExports.swift

```diff
@@ -26,3 +26,5 @@
 @_exported import struct StitchCoreSDK.UserAPIKey
 @_exported import struct StitchCoreSDK.UserAPIKeyCredential
 @_exported import struct StitchCoreSDK.UserPasswordCredential
+
+@_exported import struct MongoSwift.Document
```

Comment:
[nice] While we're at it we can also import ObjectId and any other BSON types people might find useful